manifests/qpid.pp

```diff
@@ -10,31 +10,50 @@
   String $interface = 'lo',
   Integer[0, 5000] $wcache_page_size = 4,
 ) {
-  include certs
-  include certs::qpid
   include katello::params
 
   $agent_event_queue_name = $katello::params::agent_event_queue_name
 
+  if $katello::params::enable_katello_agent {
+    include certs
+    include certs::qpid
+
+    $_qpid_ensure = 'present'
+    $ssl = true
+    $ssl_cert_db = $certs::qpid::nss_db_dir
+    $ssl_cert_password_file = $certs::qpid::nss_db_password_file
+    $ssl_cert_name = $certs::qpid::nss_cert_name
+
+    Class['certs', 'certs::qpid'] ~> Class['qpid']
+  } else {
+    $_qpid_ensure = 'absent'
+    $ssl = false
+    $ssl_cert_db = undef
+    $ssl_cert_password_file = undef
+    $ssl_cert_name = undef
+  }
```

Comment:
Introduced only including and configuring certificates when katello agent is enabled. Without this when turning it off, the `qpidd` group would not be present and cause the certificate setup to fail. This was caught by the acceptance tests.